planning/communication-plan.md

```diff
@@ -0,0 +1,56 @@
+# Communication
+
+## Group Rules
+
+<!-- any general rules you'd like to set for your group? -->
+
+Every one will be active on slack and ask questions without hesitation.
+
+---
+
+## Communication Schedule
+
+| Day     | How  | The topic of discussion |
+| ------- | :--: | ----------------------- |
+| Mon-Fri | Zoom | Work completion report  |
+
+## Communication Channels
+
+how often will we get in touch on each channel, and what we will discuss there:
+
+- **Issues**:After creating issues one can message on slack about it.
+- **Pull Requests**:After creating pull request one can inform on the slack to
+  check it.
+- **GitHub Discussions**: Any time any one can ask questions and whoever checks
+  can respond to the query
+- **Slack/Discord**: Active on Slack to ask any queries.
+- **Video Calls**: Mon-Fri (9:00 p.m)
+
+---
+
+## Availability
+
+### Availability for calling/messaging
+
+| Day |   Monday    |   Tuesday   |  Wednesday  |  Thursday   |   Friday    |  Saturday   |   Sunday    |
```

Comment:
Please add your available time for each day.

Reply:
ok @tugba will do it

Reply:
Hello @yildiztugba  do I create an issue in this repo and link this pr to it

Reply:
yes, you need an issue related to this pr and link it.

Reply:
I have created an issue @yildiztugba  and linked this pr.Please review it ?